src/tools/execution-results.h

```diff
@@ -189,10 +196,19 @@ struct ExecutionResults {
         return false;
       }
       std::cout << "[fuzz-exec] comparing " << name << '\n';
-      if (!areEqual(results[name].values, other.results[name].values)) {
-        return false;
+      if (results[name].index() != other.results[name].index()) {
+        if (ignoreTrap) {
+          if (!std::holds_alternative<Trap>(results[name]) &&
+              !std::holds_alternative<Trap>(other.results[name])) {
```

Comment:
You can also use `std::get_if` here and let the returned pointer convert to bool instead of using `std::holds_alternative` if you want to use a shorter function name.

Reply:
Done